README.md

```diff
@@ -1,2 +1,10 @@
 # WebGroceryLists
 A better way to manage shopping lists.
```

Comment:
Did you want to add a more detailed description/overview of what the webapp is designed to do?</br> 
Or some kind of roadmap for steps?

Reply:
I really should add all of those things. I'll do that this evening!